README.md

```diff
@@ -0,0 +1,23 @@
+# matlab-travis-lang-test [![Build Status](https://staging.travis-ci.org/mathworks-continuous-integration/matlab-travis-lang-test.svg?branch=master)](https://staging.travis-ci.org/mathworks-continuous-integration/matlab-travis-lang-test)
+
+Repo for testing continuous integration support for MATLAB using Travis CI.
+
+## MATLAB? Using Travis CI?
+Indeed! It all starts with a minimal `.travis.yml` that looks like the following:
+
+```yaml
+language: matlab
+```
+
+The minimal configuration will run MATLAB tests in the repo (including subfolders) to determine if they all pass.
+
+MATLAB can also be invoked directly if custom commands are desired:
```

Comment:
also can

Reply:
Let's use active voice if possible: 
You also can  run custom commands by invoking MATLAB using the -batch startup option:

Reply:
Or "...when you invoke..."

Reply:
You're right: active makes more sense here 😄